index.js

```diff
@@ -1,20 +1,45 @@
 /* jshint node: true */
-'use strict';
+"use strict";
 
-module.exports = {
-  name: 'ember-ui-sortable',
+const Funnel = require("broccoli-funnel");
+const Merge = require("broccoli-merge-trees");
+const fastbootTransform = require("fastboot-transform");
+const path = require("path");
+const existSync = require("exists-sync");
 
-  included: function(app) {
-    app.import('node_modules/jquery-ui/ui/version.js');
-    app.import('node_modules/jquery-ui/ui/data.js');
-    app.import('node_modules/jquery-ui/ui/ie.js');
-    app.import('node_modules/jquery-ui/ui/safe-active-element.js');
-    app.import('node_modules/jquery-ui/ui/safe-blur.js');
-    app.import('node_modules/jquery-ui/ui/scroll-parent.js');
-    app.import('node_modules/jquery-ui/ui/plugin.js');
-    app.import('node_modules/jquery-ui/ui/widget.js');
-    app.import('node_modules/jquery-ui/ui/widgets/mouse.js');
-    app.import('node_modules/jquery-ui/ui/widgets/draggable.js');
-    app.import('node_modules/jquery-ui/ui/widgets/sortable.js');
+ module.exports = {
+  name: "ember-ui-sortable",
+
+  treeForVendor(tree) {
+    let trees = [];
```

Comment:
The use of `let` is only necessary if the variable is reassigned later on. Mutating an object or an array isn't the same as reassigning a value to its reference. Use `const`.